openlibrary/plugins/upstream/account.py

```diff
@@ -402,6 +403,43 @@ def GET(self):
         # Show failed message without thinking.
         return render['account/verify/failed']()
 
+class account_validation(delegate.page):
+    path = '/account/validate'
+
+    @staticmethod
+    def validate_username(username):
+        if not username.isalnum():
+            return 'Username may only contain numbers and letters'
+        if not (21 > len(username) > 2):
```

Comment:
```suggestion
        if not 3 <= len(username) <= 20:
```
Allow the code to mirror the text on the next line.

Reply:
Will do it thanks :)